builders/base/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update -y && \
     rm /var/lib/apt/lists/*_*
 
 # Install the latest LTS release of nodejs
-RUN mkdir /nodejs && curl https://nodejs.org/dist/v8.12.0/node-v8.12.0-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+RUN mkdir /nodejs && curl https://nodejs.org/dist/v8.16.0/node-v8.16.0-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
```

Comment:
The comment states latest LTS.
Use this:
`https://nodejs.org/dist/v10.16.0/node-v10.16.0-linux-x64.tar.gz`

See:
https://nodejs.org/en/about/releases/